hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -807,6 +807,76 @@ def _track_subtype(self, init_values):
                 return self.env.ref("hr_timesheet_sheet.mt_timesheet_approved")
         return super()._track_subtype(init_values)
 
+    # ------------------------------------------------
+    # to prevent sending emails to the reviewer
+    # ------------------------------------------------
+
+    def get_reviewers(self, company, record_id):
+        """Get list of possible reviewers id
+
+        Args:
+            company (record): active timesheet company
+            record_id (int): active timesheet id
+
+        Returns:
+            reviewer_ids (list): list of possible reviewers id
+        """
+        # serach active timesheet
+        sheet = self.env["hr_timesheet.sheet"].search(
+            [("company_id", "=", company.id), ("id", "=", record_id)]
+        )
+        # Create list of possible reviewers
+        reviewer_ids = []
+        reviewer = sheet._get_subscribers()
```

Comment:
Why??? There is a reviewer_id field in  the model. What is the problem with using it?

Reply:
resolved